src/xcode/ENA/ENA/Source/Services/Risk/Provider/RiskProvider.swift

```diff
@@ -257,12 +263,15 @@ final class RiskProvider: RiskProviding {
 	}
 
 	private func downloadDayPackages(completion: @escaping (Result<Void, RiskProviderError>) -> Void) {
-		keyPackageDownload.startDayPackagesDownload(completion: { result in
+		keyPackageDownload.startDayPackagesDownload(completion: { [weak self] result in
 			switch result {
 			case .success:
 				completion(.success(()))
 			case .failure(let error):
-				completion(.failure(.failedKeyPackageDownload(error)))
+				// we need this delay so the user can see something happened
+				self?.targetQueue.asyncAfter(deadline: .now() + 1.0) {
```

Comment:
Don't we need this delay only in case of manual reloading?

Reply:
Delay of a second in case of an error is negligible, I don't think we should spend effort on tracking if the error happened during manual risk.